.github/workflows/robustness-nightly.yaml

```diff
@@ -27,7 +27,7 @@ jobs:
       count: 80
       testTimeout: 200m
       artifactName: main-arm64
-      runs-on: actuated-arm64-8cpu-32gb
+      runs-on: "['self-hosted', 'Linux', 'ARM64']"
```

Comment:
What about line 46?

Reply:
Lazyfs tests only run on `main` currently.

Reply:
Hmm, I think having machine consistent between test scenarios would make it clearer when debugging.